packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,625 @@
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 112,
+			"defaultValue": 0,
+			"readOnly": true
+		},
+		{
+			"#": "48",
+			"label": "Internal Sensor Temp Offset",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": -7,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "49",
+			"label": "R1 Sensor Temp Offset",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": -7,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "52",
+			"label": "Filter Timer",
+			"unit": "hours",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4000,
+			"defaultValue": 0
+		},
+		{
+			"#": "53",
+			"label": "Filter Timer Nax",
+			"unit": "hours",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4000,
+			"defaultValue": 300
+		},
+		{
+			"#": "54",
+			"label": "Heat Timer",
+			"unit": "hours",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4000,
+			"defaultValue": 0
+		},
+		{
+			"#": "55",
+			"label": "Cool Timer",
+			"unit": "hours",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 4000,
+			"defaultValue": 0
+		},
+		{
+			"#": "61",
+			"label": "Fan Purge Heat",
+			"unit": "seconds",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 90,
+			"defaultValue": 0
+		},
+		{
+			"#": "62",
+			"label": "Fan Purge Cool",
+			"unit": "seconds",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 90,
+			"defaultValue": 0
+		},
+		{
+			"#": "179",
+			"label": "Battery",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"readOnly": true
+		},
+		{
+			"#": "181",
+			"label": "Battery Stat Shutdown Threshold",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 50,
+			"defaultValue": 0
+		},
+		{
+			"#": "182",
+			"label": "Battery Radio Cutoff Threshold",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 10,
+			"maxValue": 50,
+			"defaultValue": 10
+		},
+		{
+			"#": "183",
+			"label": "Battery LOWBATT Indicator Threshold",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 20,
+			"maxValue": 75,
+			"defaultValue": 20
+		},
+		{
+			"#": "184",
+			"label": "Battery Value for Midlevel",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 50,
+			"maxValue": 80,
+			"defaultValue": 50
+		},
+		{
+			"#": "185",
+			"label": "Battery Report Timer",
+			"unit": "days",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "186",
+			"label": "Temperature Delta Autosend Threshold",
+			"description": "Required change to trigger update",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 5,
+			"defaultValue": 2
+		},
+		{
+			"#": "187",
+			"label": "Periodic Temperature Send Interval",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 120,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
```

Comment:
We usually call these parameters something like _Temperature Report Threshold_ and _Temperature Report Interval_

Reply:
Done